docker-compose.yml

```diff
@@ -39,8 +38,10 @@ services:
     image: bpanel/bcoin
     restart: unless-stopped
     env_file:
-      - ./bcoin.env
+    #   - ./bcoin.env
       - ./secrets.env
+    environment:
+      BCOIN_NETWORK: regtest
```

Comment:
This needs to be passed by env or argv for some reason otherwise it doesn't get caught in time by `new FullNode`